scripts/anonymise-data.js

```diff
@@ -59,9 +61,9 @@ parser.on('readable', async () => {
     }
     // SBI,AGREEMENT_CODE,AGREEMENT_DESC,SCHEME_OPTION,SCHEME_OPTION_DESC,DURATION,AGREEMENT_START_YR,AGREEMENT_END_YR,AGREEMENT_START_DT,AGREEMENT_END_DT,PARCEL_ID,PARCEL,HECTARES
     if (processed > 1) {
-      const [fakeSbi, fakeAgreement] = getFakeSbi(record[0])
-      record[0] = fakeSbi
-      record[1] = record[1].replace(/\d*(\/.*)*/, `${fakeAgreement}$1`)
+      const [fakeSbiId, fakeAgreementId] = getFakeIdentifiers(record[0])
```

Comment:
I think `SbiId` reads oddly, given that the `i` in `Sbi` stands for `identifier`. And from the CSV headers it should be `AgreementCode` rather than `AgreementId`.

```suggestion
      const [fakeSbi, fakeAgreementCode] = getFakeIdentifiers(record[0])
```